resolver/oneInchmulti.js

```diff
@@ -0,0 +1,28 @@
+const fetch = require("node-fetch");
+const axios = require("axios");
+
+const chainId = 56;
+const apiBaseUrl = "https://api.1inch.io/v4.0/" + chainId;
+function apiRequestUrl(methodName, queryParams) {
+  return (
+    apiBaseUrl + methodName + "?" + new URLSearchParams(queryParams).toString()
+  );
+}
+async function buildTxForQuote(quoteParams) {
+  const url = apiRequestUrl("/quote", quoteParams);
+
+  await axios
+    .get(url)
+    .then(async function (response) {
+      console.log(response.data);
+      return await response.data;
+    })
+    .catch(function (error) {
+      console.log(error);
+      return error;
+    });
+}
+async function ExecuteQuote(quoteParams) {
+  await buildTxForQuote(quoteParams);
+}
```

Comment:
No need to complicate things here. You can do all this in one function and less than ten lines of code.

Reply:
Done